### PR TITLE
done

### PR DESCRIPTION
**Pull request by VadymKapeliusnyi**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -1 +1,17 @@
 /* styles go here */
+/* label {
+  display:block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+} */
```
Comment (TarasHoliuk) [inline src/style.css]: don't leave comments

File: src/style.css [hunk 2]
```diff
@@ -1 +1,17 @@
...
+.box
+ {
+  display:block;
+  margin-top: 10px;
```
Comment (TarasHoliuk) [inline src/style.css]: it's good practice to add only bottom or only top margins

File: src/style.css [hunk 3]
```diff
@@ -1 +1,17 @@
 /* styles go here */
+/* label {
+  display:block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+} */
+.box
+ {
```
Comment (TarasHoliuk) [inline src/style.css]:
```suggestion
.box {
```

File: src/style.css [hunk 4]
```diff
@@ -1 +1,17 @@
...
+  display:block;
+  margin-top: 10px;
+  margin-bottom: 10px;
+
+ }
+.container{
```
Comment (TarasHoliuk) [inline src/style.css]:
```suggestion
.container {
```

File: src/index.html [hunk 5]
```diff
@@ -1,17 +1,141 @@
...
+          >How old are You?:<input
+            type="number"
+            id="age"
+            name="age"
+            min="1"
+            max="100"
+            value="22"
+            autocomplete="off"
+            required
+        /></label>
```
Comment (TarasHoliuk) [inline src/index.html]:
```suggestion
        <label class="box">
          How old are You?:
          
          <input
            type="number"
            id="age"
            name="age"
            min="1"
            max="100"
            value="22"
            autocomplete="off"
            required
        />
      </label>
```

fix it everywhere (proofread [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md))

- About code style: all siblings (including text) should be split by an empty line
- if there are nested tags write each one in the new line

File: src/index.html [hunk 6]
```diff
@@ -1,17 +1,141 @@
...
+        <label class="box"
+          >Would you recommend us?:<select>
+            <option value="yes">yes</option>
+            <option value="no">no</option>
+          </select></label
+        >
+      </fieldset>
+      <button type="submit" class="container">Submit</button>
+    </form>
+
```
Comment (TarasHoliuk) [inline src/index.html:191]: redundant empty line between parent and child tags

**Review by TarasHoliuk — CHANGES_REQUESTED** [review]
Make sure you passed all tests and deployed tests page
If you have any questions feel free to ask in the chat

File: src/index.html (hunk 6 above)
Comment (VadymKapeliusnyi) [reply src/index.html:191]: Good evening.
Could you check the latest fix?

**Pull request closed without merging**